DailyKit/CallManager/Models/CallMuteState.swift

```diff
@@ -0,0 +1,34 @@
+import Daily
+import AVFoundation
+
+public enum CallMuteState: Equatable {
+    case muted, unmuted
+}
+
+extension CallMuteState {
+    init(
+        _ settings: CameraInputSettings,
+        status: AVAuthorizationStatus
+    ) {
+        switch (settings.isEnabled, status) {
+        case (true, .authorized):
+            self = .unmuted
+
+        case (_, _):
+            self = .muted
+        }
+    }
+
+    init(
+        _ settings: MicrophoneInputSettings,
+        status: AVAuthorizationStatus
+    ) {
+        switch (settings.isEnabled, status) {
+        case (true, .authorized):
+            self = .unmuted
+
+        case (_, _):
```

Comment:
It's a "catch all" wildcard.

So yeah, in other languages it would be called `default`.
Swift also has a `case default:` syntactic sugar for it, but `_` is more general.
(Personal pet peeve: the `case default:` should not exist, we already have `case _:` for that)